lib/models/organisation-test.js

```diff
@@ -2,10 +2,71 @@ import chai, { expect } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { Error } from 'mongoose';
 import Organisation from './organisation';
+import User from './user';
+import Role from './role';
+import { createOrganisation, createUser } from './test-utils/testModelsHelper';
 
 chai.use(chaiAsPromised);
 
 describe('organisation model', () => {
+  afterEach(async () => {
+    await Role.deleteMany({});
+    await Organisation.deleteMany({});
+    await User.deleteMany({});
+  });
+
+  it('should properly create organisation', async () => {
+    const user = await createUser();
+    const organisation = await createOrganisation();
+
+    const userAfterCreateOrganisation = await User.findOne({
+      email: user.email
+    });
+
+    const createdUsersOrganisation = userAfterCreateOrganisation
+      .organisations
+      .find(id => id.toString() === organisation._id.toString());
+    const createdUsersOrganisationSetting = userAfterCreateOrganisation
+      .organisationSettings
+      .find(orgSetting =>
+        orgSetting.organisation.toString() === organisation._id.toString()
+      );
+    const createdUsersRoles = await Role.find({
+      organisation: organisation._id
+    });
+
+    await expect(createdUsersOrganisation).to.not.be.undefined;
+    await expect(createdUsersOrganisationSetting).to.not.be.undefined;
+    await expect(createdUsersRoles).to.be.an('array').to.have.lengthOf(2);
```

Comment:
I guess it would be nice also to check those roles for proper structure and names, etc.

Reply:
But wait, in this case it should be placed into `User`'s model tests, so nevermind